enatega-multivendor-app/App.js

```diff
@@ -195,33 +195,23 @@ export default function App() {
     registerForPushNotificationsAsync()
 
     notificationListener.current = Notifications.addNotificationReceivedListener(notification => {
-      console.log('addNotificationReceivedListener', notification?.request?.content?.data)
-      try {
-      if (notification?.request?.content?.data?.type === NOTIFICATION_TYPES.REVIEW_ORDER) {
-        const id = notification?.request?.content?.data?._id
+      if (notification?.request?.trigger?.remoteMessage?.data?.type === NOTIFICATION_TYPES.REVIEW_ORDER) {
+        const id = notification?.request?.trigger?.remoteMessage?.data?._id
         if (id) {
           setOrderId(id)
           reviewModalRef.current.open()
         }
       }
-      } catch (error) {
-      console.log('addNotificationReceivedListener', error) 
-    }
     })
 
     responseListener.current = Notifications.addNotificationResponseReceivedListener(response => {
-try {
-      console.log('addNotificationResponseReceivedListener', response?.notification?.request?.content?.data)
-      if (response?.notification?.request?.content?.data?.type === NOTIFICATION_TYPES.REVIEW_ORDER) {
-        const id = response?.notification?.request?.content?.data?._id
-        if (id){
+      if (response?.notification?.request?.trigger?.remoteMessage?.data?.type === NOTIFICATION_TYPES.REVIEW_ORDER) {
+        const id = response?.notification?.request?.trigger?.remoteMessage?.data?._id
+        if (id) {
           setOrderId(id)
           reviewModalRef.current.open()
         }
       }
-    } catch (error) {
```

Comment:
what's the purpose of removing try/catch block here ?

Reply:
This was used By Senior Adnan just for Testing Notification

Reply:
Ok merging it now, but it's better to keep try/catch as it prevents app to break.